DataFormats/CTPPSDigi/src/CTPPSPixelDataError.cc

```diff
@@ -0,0 +1,98 @@
+
+#include "DataFormats/CTPPSDigi/interface/CTPPSPixelDataError.h"
+
+
+CTPPSPixelDataError::CTPPSPixelDataError() {}
+
```

Comment:
please make sure that all basic type data members are initialized or delete this default constructor

Reply:
done